spec/defines/define_plugin_spec.rb

```diff
@@ -7,17 +9,19 @@
         facts
       end
 
-      let(:pre_condition) { %q(
+      let(:pre_condition) do
+        '
```

Comment:
For multi line strings I'd suggest [heredoc](https://www.rubyguides.com/2018/11/ruby-heredoc/)
```ruby
<<~PUPPET
include elastic_stack::repo
include logstash
PUPPET
```